### PR TITLE
Stubs 4

### PR DESCRIPTION
**Pull request by CandyCrystal**
I believe this should be the rest of the stubs all done!

### CODE REVIEW

**Comment by CandyCrystal** [comment]
That should be it all fixed!

File: palace_of_winds.logic [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  room F4EastWindGap
+  room F4KeyMiniboss
+  room F4NorthEastChest
+  room F4Stairs
+  room F4UnstableFloor
+  room F4WestExterior
+  room F4WestWindGap
+  room GFBlockPush
+  room GFBridges
+  room GFCapeMiniboss
```
Comment (Ibot02) [inline palace_of_winds.logic:46]: I think I'd prefer a name not based on vanilla placements where possible.
```suggestion
  room GFWizrobesMiniboss
```

File: town_minish_houses.logic [hunk 2]
```diff
@@ -0,0 +1,10 @@
+area TowninishHouses:
```
Comment (Ibot02) [inline town_minish_houses.logic:1]: Probably a typo?
```suggestion
area TownMinishHouses:
```

File: well.logic [hunk 3]
```diff
@@ -0,0 +1,3 @@
+area Well:
+  room Main
+  room WellEntry
```
Comment (Ibot02) [inline well.logic:3]: I think
```suggestion
  room SouthEntry
```
is clearer?

**Review by Ibot02 — APPROVED** [review]

File: castle_courtyard.logic [hunk 4]
```diff
@@ -0,0 +1,2 @@
+area CastleCourtyard:
+  room Main
```
Comment (Ibot02) [inline castle_courtyard.logic:2]: Isn't this already included? In hyrule_castle_garden.logic

**Review by Ibot02 — CHANGES_REQUESTED** [review]

**Pull request merged**